pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/BookkeeperSchemaStorage.java

```diff
@@ -237,9 +237,7 @@ public SchemaVersion versionFromBytes(byte[] version) {
 
     @Override
     public void close() throws Exception {
-        if (nonNull(bookKeeper)) {
-            bookKeeper.close();
```

Comment:
I would keep this call to close()

we cannot make assumptions on what the BK client does on `close()`

Reply:
This was problem with the rebase. the BK client instance is owned and must be closed here.